soccer/planning/LineKickPlanner.cpp

```diff
@@ -0,0 +1,201 @@
+#include "LineKickPlanner.hpp"
```

Comment:
@ashaw596 can you add some comments to this file? (at least overall function descriptions?)